tesler-doc-db/src/main/resources/db/migration/liquibase/data/latest/widgets/navigation/navigation-tabs.md

```diff
@@ -1,2 +1,72 @@
 # Tabs
+Tabs are needed to navigate at screen level between different views. They have their own hierarchy: first-level menu tabs are seen always on the current screen, second-level tabs are seen when you choose a tab from first-level menu, etc.
 
+For example, on the current page you can see first-level menu (*"Navigation"*, *"Widgets"*, *"Fields"*) and current second-level menu (*"Overview"*, *"Screen"*, ...). If you choose another first-level tab, you will see another set of second-level tabs. 
+
+More info about level-menu hierarchy you can find at *[Major Concepts](#/screen/getting-started/view/screen/)*
+
+## How to create?
+Tabs are set in **screen.json** files. First-level tabs are in **menu** block. It can have only a view or some child-views. For example:
+```ts
+{
+  "name": "example",
+  "title": "Example Title",
+  "primaryViewName": "no-child-view",
+  "primaryViews": [
+    "no-child-view"
+  ],
+
+  "_comment": "Primary view is a view set by default;"
+
+  "navigation": {
+    "menu": [
+      {
+        "viewName": "no-child-view"
+      },
+      {
+        "title": "View with children",
+        "child": [
+
+        "_comment": "Here comes second-level tabs"
+
+            { "viewName": "first-child" },
+            { "viewName": "second-child" },
+    
+            "_comment": "children can also have their own children, making third-level menu and so on"
+        ]
+      }
+    ]
+  }
+}
+```
+To provide second-level menu, a view must have **SecondLevelMenu.widget.json**:
+```ts
+{
+  "name" : "SecondLevelMenu",
+  "title" : "Navigation",
+  "type" : "SecondLevelMenu",
+  "showCondition" : [ ],
+  "fields" : [ ],
+  "axisFields" : [ ],
```

Comment:
This is not a required field, so as `showConditions` and the `chart`.
Check the schema 
https://github.com/tesler-platform/tesler-schema/blob/572a65079df1f30e748a1d824e54a2fa46d2d839/src/files/WidgetMeta.ts#L25